src/syft/common/lazy_structures.py

```diff
@@ -0,0 +1,171 @@
+from typing import Callable
+
+
+class LazyDict:
```

Comment:
Q: The ```dict``` has also an ```update``` method. This is expected to behave like a real dictionary only that the ```get``` method has a different behavior?